README.md

```diff
@@ -12,219 +12,234 @@
 </p>
 <br>
 
-### Contents
-------
-+ Models
-    - [GLM](#glm)
-    - [DESM](#desm)
-    - [DSSM](#dssm)
-    - [CDSSM](#cdssm)
-    - [LSTM-DSSM](#lstm-dssm)
-    - [ARC-I](#arc-i)
-    - [ARC-II](#arc-ii)
-    - [MultiGranCNN](#multigrancnn)
-    - [MV-LSTM](#mv-lstm)
-    - [ABCNN](#abcnn)
-    - [DRMM](#drmm)
-    - [K-NRM](#k-nrm)
-    - [CONV-KNRM](#conv-knrm)
-    - [aNMM](#anmm)
-    - [DUET](#duet)
-    - [MatchPyramid](#matchpyramid)
-    - [match-LSTM](#match-lstm)
-    - [DRMM-TKS](#drmm-tks)
-    - [BiMPM](#bimpm)
-    - [DeepRank](#deeprank)
-    - [Match-SRNN](#match-srnn)
-    - [Co-PACRR](#co-pacrr)
-    - [DLCM](#dlcm)
-    - [Siamese-LSTM](#siamese-lstm)
-    - [ESIM](#esim)
-    - [MCAN](#mcan)
-    - [MIX](#mix)
-    - [HiNT](#hint)
+Text matching is a core component in many natural language processing tasks, where many task can be viewed as a matching between two texts input.
 
+​							$$\text{Match}(s, t) = g(f(\psi(s), \phi(t)))$$
 
-### Models
-------
+Where $s$ and $t$ are source text input and target text input, respectively. The $\psi$ and $\phi$ are representation function for input $s$ and $t$, respectively. The $f$ is the interaction function, and $g$ is the aggregation function. The representative matching tasks are as follows:
```

Comment:
Can we add a link of the survey on the neural ranking models here to further explain this formula ? [Survey](https://arxiv.org/abs/1903.06902)

Reply:
Good~